modules/04_PE_FE_parameters/iea2014/datainput.gms

```diff
@@ -245,12 +250,10 @@ loop(regi,
      );
 );
 
+***------------------ allocate own power consumption to electricity technologies -----------------------------
 *RP 2019-02-19: This is now changed starting from rev 8352. Power plant output is from now on gross production instead of net, and power plant autoconsumption is shifted to t&d losses
 *RP This was done to facilitate comparison with other sources which usually report gross electricity generation as well as gross capacity factors
-***------------------ allocate own power consumption to electricity technologies -----------------------------
-***p04_IO_output(regi,enty,enty2,te)$(sameas(enty2,"seel") AND (NOT sameas(te,"wind")) AND (NOT sameas(te,"spv")) )  = p04_IO_output(regi,enty,enty2,te)
-***                                                           - ( p04_IO_output(regi,enty,enty2,te) / sum(pe2se(enty3,enty2,te2)$((NOT sameas(te2,"wind")) AND (NOT sameas(te2,"spv"))), p04_IO_output(regi,enty3,enty2,te2)) ) 
-***                                                             * f04_IO_output("2005",regi,"seel","feel","o_feel");
+
```

Comment:
why is the output power rescaling removed? 
or was it just shifted somewhere and I didn't find it?

If I am not mistaken it is important to have it here, otherwise we will be back at net production.

Reply:
ok, I see. 

what about removing all these lines here (including the header "allocate own power consumption to electricity technologies", and instead adding above in line 38 (so when f04_IO_output is read in) the following comment: 

*** RP 2019-02-19: From rev 8352 on, f04_IO_output contains gross generation for power plants. Power plant autoconsumption is contained in t&d losses. This was done to facilitate comparison with other sources which usually report gross electricity generation as well as gross capacity factors